azurelinuxagent/common/telemetryevent.py

```diff
@@ -83,51 +83,51 @@ def __eq__(self, other):
 
 
 # too-few-public-methods<R0903> Disabled: This class is used as an Enum
-class TelemetryEventPriorities(object): # pylint: disable=R0903
-    """
-    Class defining the priorities for telemetry events. Lower the number, higher the priority
-
-    Note: 0 is reserved for a feature like QuickLog in the Windows Agent (i.e. the ability to send out telemetry
-    instantly rather than waiting for a minute for the monitor thread to pick up the events)
-    """
-    AGENT_EVENT = 1 # Agent events always get the highest priority
-    EXTENSION_EVENT_NEW_PIPELINE = 2    # Prioritize extensions using the new dedicated pipeline over extensions hijacking the agent pipeline
-    EXTENSION_EVENT_OLD_PIPELINE = 3
+# class TelemetryEventPriorities(object): # pylint: disable=R0903
```

Comment:
Remove?

Reply:
Same for the rest of the code regarding prioritization that's commented out below.

Reply:
Ohh shoot forgot to cleanup! Thanks for catching this, will clean this up

Reply:
Done!